src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package frc.team2412.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import static frc.team2412.robot.Hardware.*;
+
+public class IntakeSubsystem extends SubsystemBase{
+	public static final double INTAKE_IN_SPEED = 0.3;
+	public static final double INTAKE_FAST_OUT_SPEED = -0.7;
+
+    CANSparkMax intakeMotorBack;
+    CANSparkMax intakeMotorRight;
+    CANSparkMax intakeMotorFront;
+    CANSparkMax intakeMotorLeft;
+
+    CANSparkMax conveyorBeltOne;
+    CANSparkMax conveyorBeltTwo;
+
+    public IntakeSubsystem() {
+
+        intakeMotorBack = new CANSparkMax(intakeMotorBackID, MotorType.kBrushless);
+        intakeMotorRight = new CANSparkMax(intakeMotorRightID, MotorType.kBrushless);
+        intakeMotorFront = new CANSparkMax(intakeMotorFrontID, MotorType.kBrushless);
+        intakeMotorLeft = new CANSparkMax(intakeMotorLeftID, MotorType.kBrushless);
+
+        conveyorBeltOne = new CANSparkMax(conveyorBeltOneID, MotorType.kBrushless);
+        conveyorBeltTwo = new CANSparkMax(conveyorBeltTwoID, MotorType.kBrushless);
+
+        intakeMotorBack.follow(intakeMotorFront);
+        intakeMotorLeft.follow(intakeMotorFront);
+        intakeMotorRight.follow(intakeMotorFront);
+
+    }
+    
+    public void intake(double speed, CANSparkMax motor){
+        motor.set(speed);
```

Comment:
This needs to use one of the class member variables, as nothing outside this class should have access to have a CANSparkMax instance.